widget/hyperlink.go

```diff
@@ -94,7 +94,7 @@ func (hl *Hyperlink) textStyle() fyne.TextStyle {
 
 // textColor tells the rendering textProvider our color
 func (hl *Hyperlink) textColor() color.Color {
-	return theme.HyperlinkColor()
```

Comment:
This is going to break custom themes.
We deprecated the Hyperlink color for a reason, we can't really change it until 2.0

Reply:
Okay, then I will remove the deprecated color from the test theme and apply the same delegation as for the internal themes.